### PR TITLE
Added ship detector notebook

### PR DESCRIPTION
**Pull request by nateweisiger**

### CODE REVIEW

File: jupyter-notebooks/ship-detector/01_ship_detector.ipynb [hunk 1]
```diff
@@ -0,0 +1,304 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Detect ships in Planet data\n",
+    "This notebook demonstrates how to use Plant data to detect and count the number of ships using image data clipped from an AOI (in this case, a small area in the San Francisco Bay).\n",
```
Comment (danabauer) [inline jupyter-notebooks/ship-detector/01_ship_detector.ipynb]: change Plant to Planet

File: jupyter-notebooks/ship-detector/01_ship_detector.ipynb [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+    "\n",
+    "Once ships are detected, the script will generate pngs of the ships and a png of input the image with outlines around the ships."
+   ]
```
Comment (danabauer) [inline jupyter-notebooks/ship-detector/01_ship_detector.ipynb]: Suggestion: Change opening paragraph to "This notebook demonstrates how to detect and count objects in satellite imagery using algorithms from Python's [scikit-image](http://scikit-image.org/) library. In this example, we'll look for ships in a small area in the San Francisco Bay and generate a PNG of each ship with an outline around it.

File: jupyter-notebooks/ship-detector/01_ship_detector.ipynb [hunk 3]
```diff
@@ -0,0 +1,304 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Input Parameters\n",
+    "This is a sample image that was generated using the Clip and Ship API. To test this with your own image, replace the parameters below."
+   ]
```
Comment (danabauer) [inline jupyter-notebooks/ship-detector/01_ship_detector.ipynb]: link to https://www.planet.com/docs/reference/clips-api/

File: jupyter-notebooks/ship-detector/01_ship_detector.ipynb (hunk 1 above)
Comment (nateweisiger) [inline jupyter-notebooks/ship-detector/01_ship_detector.ipynb]: Done

File: jupyter-notebooks/ship-detector/01_ship_detector.ipynb (hunk 2 above)
Comment (nateweisiger) [inline jupyter-notebooks/ship-detector/01_ship_detector.ipynb]: That is better. Done.

File: jupyter-notebooks/ship-detector/01_ship_detector.ipynb (hunk 3 above)
Comment (nateweisiger) [inline jupyter-notebooks/ship-detector/01_ship_detector.ipynb]: Done

**Pull request merged**